README.md

```diff
@@ -1,5 +1,10 @@
 # Jujutsu VCS
 
+**Documentation:**
+[docs for the latest release](https://martinvonz.github.io/jj/latest/),
+[docs for the prerelease (main branch) version](https://martinvonz.github.io/jj/prerelease/).
+
+
```

Comment:
What do you think about putting these links under "Getting started"? It feels like that's the point where the docs are useful. If we link to them from here, it seems like we should move most of the rest of this docs into the docs.

Reply:
This seems like a good idea. Alternatively, I'm still thinking of getting rid of them entirely.

In my mind, their main purpose was to inform users that there are *two* documentation websites.

Reply:
I got rid of them and rewrote the "Getting started" section a bit. PTAL.

Reply:
Looks good, thanks!